test/src/apps/devfs/framebuffer.c

```diff
@@ -0,0 +1,193 @@
+#define _GNU_SOURCE
+
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
I am not aware of a case in which we placed the license line after the `#define _GNU_SOURCE` line.

https://github.com/asterinas/asterinas/blob/2034055f90607f67f04de044ee88dd52a488d612/test/src/apps/clone3/clone_process.c#L1-L4

Prefer to follow the style of other tests, see: https://github.com/search?q=repo%3Aasterinas%2Fasterinas+_GNU_SOURCE&type=code&p=1